spec/linter-stylelint-spec.js

```diff
@@ -301,4 +303,33 @@ describe('The stylelint provider for Linter', () => {
       );
     });
   });
+
+  describe('works with JS files (language-babel) with Styled Components and', () => {
+    beforeEach(() => {
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
+      waitsForPromise(() => atom.packages.activatePackage('language-babel'));
+    });
+
+    it('correctly detects an error present in a file', () => {
+      const ntscMessage = 'Expected a trailing semicolon (<a href="http://stylelint.io/user-guide/rules/declaration-block-trailing-semicolon">declaration-block-trailing-semicolon</a>)';
```

Comment:
I'm not sure why ESLint isn't blowing up on this line being too long already, but can you split this into two lines?